semeio/jobs/design_kw/design_kw.py

```diff
@@ -116,3 +119,55 @@ def extract_key_value(parameters):
     if errors:
         raise ValueError("\n".join(errors))
     return res
+
+
+def rm_genkw_prefix(paramsdict, ignoreprefixes="LOG10_"):
+    """Remove anything before the first colon in a string, or remove
```

Comment:
The doc string still need an update, besides that it looks good 👍 Nice clean up of `rm_genkw_prefix` ;)

Reply:
Updated.